scripts/updateCredentials.js

```diff
@@ -0,0 +1,143 @@
+/*!
+Copyright 2019 RtF-US
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+
+// This script updates a client credentials with the provided JSON data:
+//
+// Usage: node scripts/updateCredentials.js CouchDB-url clientCredentialId filename
+// @param {String} CouchDB-url - The url to the CouchDB where docoments should be verified.
+// @param {String} clientCredentialId - The "_id" value of the client credentials.
+// @param {String} filename - The path to a file that contains the JSON data to use for the update.
```

Comment:
This doc should also explain that the JSON can only contain fields/values that need to be updated. It is not necessary to provide a complete structure of a client credential record.

In the script, verify:
1. If "_id" is given in the file, either make sure its value matches the value of the second parameter `clientCredentialId`, or don't allow "_id" to be given in the file at all;
2. JSON in this file should not contain "_revision" field.